go.mod

```
@@ -2,4 +2,7 @@ module github.com/BESTSELLER/terraform-provider-netbox
 
 go 1.14
```

Maybe upgrade to 1.15?
```suggestion
go 1.15
```

---

updated